io.podman_desktop.PodmanDesktop.yml

```diff
@@ -0,0 +1,98 @@
+app-id: io.podman_desktop.PodmanDesktop
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "21.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+command: run.sh
+separate-locales: false
+finish-args:
+  - "--socket=session-bus"
+  - "--socket=wayland"
```

Comment:
does it support wayland out of the box?

Reply:
wayland is supported with Electron yes (but there is still a fallback to x11 if system is not providing wayland)